docs/contribution-guide/releasing.rst

```diff
@@ -37,11 +37,63 @@ Manual Process (reference only)
 
 
+RPM Builds
+------------
+
+Once the GitHub release is complete RPM builds for Fedora and EPEL can be performed.
+
+A pre-requisite for this is that the developer running the updates has a `Fedora Account System <https://fedoraproject.org/wiki/Account_System?rd=Infrastructure/AccountSystem>`__ account,
+has Kerberos authentication setup as described `here <https://fedoraproject.org/wiki/Infrastructure/Kerberos>`__ and has correctly
+setup `Pagure <https://docs.pagure.org/pagure/usage/>`__  with a SSH key `here <https://pagure.io/settings#nav-ssh-tab>`__.
+
+Ensure the following RPM packages are installed on your system:
+
+* fedora-packager
+* fedora-packager-kerberos
+* krb5-workstation
+* rpmdevtools
+* fedpkg
+
+The CEKit RPM repository is stored on Pagure at https://src.fedoraproject.org/rpms/cekit
+
+Unfortunately several manual steps must be done next:
+
+1. Use ``fkinit -u <user-id>`` to create your Kerberos principal. Note that by ``klist`` will only show the last created, so use ``klist -A`` to show all.
+2. Switch to the main branch first (which is an alias for Rawhide).
+3. Update the spec file with the latest release. Note that the ``Release`` version should be reset to zero for the next command. For example, the changes might be:
+
+.. code-block:: diff
+
+   Name:           %{modname}
+  -Version:        3.9.0
+  -Release:        3%{?dist}
+  +Version:        3.12.0
+  +Release:        0%{?dist}
+   Summary:        Container image creation tool
+   License:        MIT
+   URL:            https://cekit.io
+  -Source0:        https://github.com/cekit/cekit/archive/%{release_version}/%{name}-%{release_version}.tar.gz
+  +Source0:        https://github.com/cekit/cekit/archive/refs/tags/%{version}.tar.gz
+
+
+
+4. Run ``spectool -g cekit.spec`` which will parse the ``Source0`` and download the sources.
+5. Run ``rpmdev-bumpspec cekit.spec -c "Release 3.12.0"`` which will update the ``Release`` version and generate a change log entry.
+6. Run ``fedpkg new-sources *tar.gz`` to upload the new version. Note that you can explicitly specify the tar file and the system won't re-upload any old tar files. The lookaside cache applies to all branches so only a single upload is required.
+7. Then commit and push the changes.
+8. Run a build via ``fedpkg build --scratch``
+9. If that succeeds then run ``fedpkg build`` (or ``fedpkg build --nowait``)
+
+Then for each branch active in https://src.fedoraproject.org/rpms/cekit/branches switch branch (with ``fedpkg switch-branch f33``) and repeat steps 3->9. It *may* be possible to simply cherry-pick the commit from step 7, then perform steps 8 and 9.
+
+Finally using https://bodhi.fedoraproject.org/updates/?packages=cekit submit the packages as updates. Once the packages have reached appropriate karma or the requisite time has passed they may be pushed.
```

Comment:
@goldmann Only bit I'm not entirely clear on. And did you saw Rawhide updates don't have to be added as updates?

Reply:
Correct, this is done automatically for you once the build is done.